src/ui/common/src/components/workflows/StatusBar.tsx

```diff
@@ -124,11 +124,11 @@ const ActiveWorkflowStatusTab: React.FC<ActiveWorkflowStatusTabProps> = ({
     if (!label) {
       if (nodeType.includes('Op')) {
         return 'Operator';
+      } else if (nodeType.includes('Artifact')) {
+        return 'Artifact';
       }
 
-      // otherwise, it's an Artifact
-      // nodeType will be "tableArtifact", etc.
-      return 'Artifact';
+      return '';
```

Comment:
fix is here. nodeType comes back as '' for empty items, so just return empty string here.